docs/output.md

```diff
@@ -215,6 +215,21 @@ If the pipeline is run with phasing, the aligned reads will be happlotagged usin
 | `snvs/family/{family}/{family}_snv_annotated_ranked.vcf.gz`     | VCF file with annotated and ranked variants per family   |
 | `snvs/family/{family}/{family}_snv_annotated_ranked.vcf.gz.tbi` | Index of the ranked VCF file                             |
 
+[filter_vep](https://www.ensembl.org/vep) and [bcftools view](https://samtools.github.io/bcftools/bcftools.html) can be used to filter variants.
+
+!!!note
+
+    Variants are only output if either of `--filter_variants_hgnc_id` and `--filter_snvs_expression` has been used, and only project variants are output.
+
+!!!tip
+
+    Filtered variants are output alongside unfiltered variants as additional files.
+
+| Path                                             | Description                                   |
+| ------------------------------------------------ | --------------------------------------------- |
+| `snvs/{project}/{project}_*_filtered.vcf.gz`     | VCF file with filtered variants for a project |
+| `snvs/{project}/{project}_*_filtered.vcf.gz.tbi` | Index of the filtered VCF file                |
```

Comment:
any reason for using `project` here and `family` for SVs?

Reply:
No, I will update to family here. It's a miss.